website/docs/intro/index.mdx

```diff
@@ -6,9 +6,9 @@ description: |-
 
 # What is OpenTofu?
 
-OpenTofu is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTofu can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
+OpenTofu is an infrastructure as code tool that lets you define both cloud and on-premises resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTofu can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
 
-## How does OpenTofu work?
+## How does OpenTofu works?
```

Comment:
The original form was grammatically correct for English.  Consider reverting this portion of the change.